pinot-integration-tests/pom.xml

```diff
@@ -249,5 +249,16 @@
       <artifactId>annotations</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-core_2.12</artifactId>
+          <version>3.5.1</version>
```

Comment:
is this version needed? nothing else seems to specify a version

Reply:
you're right not needed, I'll see if I can avoid adding this and if it can fetch it through the `pinot-batch-ingestion-spark-3` dep, or explicitly define the version as a property like `pinot-batch-ingestion-spark-3` does